--- src/private/ed25519.rs
+++ src/private/ed25519.rs
@@ -17,7 +17,7 @@ use subtle::{Choice, ConstantTimeEq};
 /// Ed25519 private key.
 // TODO(tarcieri): use `ed25519::PrivateKey`? (doesn't exist yet)
 #[derive(Clone)]
-pub struct Ed25519PrivateKey([u8; Self::BYTE_SIZE]);
+pub struct Ed25519PrivateKey(pub [u8; Self::BYTE_SIZE]);
 
 impl Ed25519PrivateKey {
     /// Size of an Ed25519 private key in bytes.
